### PR TITLE
chore: add a client helper script

### PR DESCRIPTION
**Pull request by Charles546**
#### Description

The script can be used to interact with honeydipper through API
or webhook.

#### This PR fixes the following issues
N/A

### CODE REVIEW

**Comment by Charles546** [comment]
Not sure if this should live here, maybe in `honeydipper-config-essentials`?

**Comment by wyardley** [comment]
> Not sure if this should live here, maybe in honeydipper-config-essentials?

Impressive, but IMO, this should probably be a go program, and the commit should be `feat` vs `chore`. Then the go program can be built as a `dipperclient` Docker image.

File: hdclient.sh [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  export HD_WEBHOOK_STATUS_CODE="$(curl -s -o "$HD_WEBHOOK_RETURN" -w "%{http_code}" -H "Token: $HD_WEBHOOK_TOKEN" "$urlprefix/$hook?accept_uuid")"
+  local ret="$?"
+
+  if [[ "$HD_WEBHOOK_STATUS_CODE" != "200" ]]; then
+    echo got status code "$HD_WEBHOOK_STATUS_CODE" >&2
+    return 1
+  fi
+
+  export HD_EVENT_ID="$(cat "$HD_WEBHOOK_RETURN" | jq -r ".eventID")"
```
Comment (wyardley) [inline hdclient.sh:128]: IMO, this shouldn't be a shell script, but if it is, you need to test for `jq` and `curl` being present.

**Comment by Charles546** [comment]
Yes, probably eventually it will be a go program. For now, just a stop gap arrangement so we don't have to deal with creating tests, build/release process, making different binaries for MacOS, Linux and some other OSes.

Good point on testing the `curl` and `jq`.

I am thinking of putting this in a separate repo instead of a byproduct of the daemon.

**Pull request closed without merging**